src/mlpack/methods/ann/layer/softshrink_impl.hpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * @file softshrink.hpp
+ * @author Lakshya Ojha
+ *
+ * Definition of Soft Shrink activation function
```

Comment:
This is the implementation.

Reply:
Sorry for the ignorance. Have corrected the same in new commit.
Thanks.